Formula/strace.rb

```diff
@@ -8,14 +8,14 @@ class Strace < Formula
     sha256 "a88334dff8142b5b45ec73cdbb702946ad7c890be7a40b4e6baa2c25a620eb50" => :x86_64_linux
   end
 
-  depends_on "linux-headers"
-
   head do
     url "https://github.com/strace/strace.git"
     depends_on "autoconf" => :build
     depends_on "automake" => :build
   end
 
+  depends_on "linux-headers"
```

Comment:
Is is a run-time dependency. At least, paths are hard-coded into the `strace` binary
```
$ strings /home/linuxbrew/.linuxbrew/Cellar/strace/4.17/bin/strace | grep linux
...
/home/linuxbrew/.linuxbrew/include/asm-generic
...
$ ls -l /home/linuxbrew/.linuxbrew/include/asm-generic
lrwxrwxrwx 1 linuxbrew linuxbrew 51 Jul  5 21:26 /home/linuxbrew/.linuxbrew/include/asm-generic -> ../Cellar/linux-headers/3.18.27/include/asm-generic
```